src/main/play-doc/operation/Dcos.md

```diff
@@ -0,0 +1,87 @@
+# DC/OS Setup
+
+The following guide will outline the steps to deploy and run ConductR as a framework within DC/OS.
+
+## Prerequisite
+
+* An existing, working DC/OS cluster. The installation and setup of DC/OS cluster is outside of the scope of this guide.
+* A working installation of Marathon in the DC/OS cluster is required to deploy ConductR into the DC/OS cluster.
+* A working installation of DC/OS CLI tools successfully authenticated against the DC/OS cluster.
+* A working bastion host with access to the DC/OS cluster's network is required to securely load and run ConductR bundles.
+
+
+## Bastion host setup
+
+Install the [[ConductR CLI|CLI]] in the bastion host to securely load and run ConductR bundles.
+
+
+## Deploy ConductR into DC/OS cluster
+
+Obtain the ConductR's application definition JSON from the [[download page|https://www.lightbend.com/product/conductr/download]] and deploy the application definition through the Marathon UI.
+
```

Comment:
Post the json to Marathon. This can be done using json mode of the 'create application' dialog. Refer to...